src/components/report/ReportAction.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import { Box, Typography, Button } from '@mui/material'
+import { makeStyles } from 'tss-react/mui'
+import { useNavigate } from 'react-router'
+import { CommonModal } from '../commonModal/CommonModal'
+import theme from '../../styles/createTheme'
+
+const ReportAction = () => {
+  const { classes } = useStyles()
+  const navigate = useNavigate()
+  const handleGoBack = () => {
+    navigate(-1)
+  }
+
+  const handleBlock = () => {
+    alert('handle block')
```

Comment:
just to show that there should be user blocking

Reply:
From my point of view, it's not quite clear when you use `alert` for this purpose. I think something like this would be better `// TODO: Add API and code for user blocking`

Reply:
Done.